libgoods/model_fetch.py

```diff
@@ -126,6 +126,62 @@ def get_bounds(model_name: str) -> Tuple[float, float, float, float]:
     return main_cat[model_name].metadata["bounding_box"]
 
 
+def is_monotonic(data: np.array) -> bool:
+    """Return true if the array is monotonically increasing or decreasing."""
+    increasing = np.all(data[1:] >= data[:-1])
+    decreasing = np.all(data[1:] <= data[:-1])
+    return increasing or decreasing
```

Comment:
not a big deal, but I think this could take advantage of short-cutting:

`return np.all(data[1:] >= data[:-1]) or np.all(data[1:] <= data[:-1])`